build/fuse-clients/ossfs/install-ossfs.sh

```diff
@@ -4,7 +4,7 @@ echo "TARGETPLATFORM: $TARGETPLATFORM"
 echo "installing ossfs"
 case $TARGETPLATFORM in
     linux/amd64)
-        yum install -y https://ack-csiplugin.oss-cn-hangzhou.aliyuncs.com/pre/ossfs/ossfs_1.88.3_centos8.0_x86_64.rpm
+        yum install -y https://ack-csiplugin.oss-cn-hangzhou.aliyuncs.com/pre/ossfs/ossfs_1.88.4_centos8.0_x86_64.rpm
```

Comment:
we are not going to support metrics on arm64 platform?

Reply:
We've never supported ossfs metrics on arm64 ever. It will be available after ossfs rebasing 1.91 version.